[2025-05-27T05:47:10.002+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: walmart_etl_dag.generate_synthetic_data manual__2025-05-27T05:47:08+00:00 [queued]>
[2025-05-27T05:47:10.010+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: walmart_etl_dag.generate_synthetic_data manual__2025-05-27T05:47:08+00:00 [queued]>
[2025-05-27T05:47:10.011+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-27T05:47:10.025+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): generate_synthetic_data> on 2025-05-27 05:47:08+00:00
[2025-05-27T05:47:10.036+0000] {standard_task_runner.py:57} INFO - Started process 636 to run task
[2025-05-27T05:47:10.040+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'walmart_etl_dag', 'generate_synthetic_data', 'manual__2025-05-27T05:47:08+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/walmart_etl_dag.py', '--cfg-path', '/tmp/tmpo4asd1gg']
[2025-05-27T05:47:10.041+0000] {standard_task_runner.py:85} INFO - Job 66: Subtask generate_synthetic_data
[2025-05-27T05:47:10.093+0000] {task_command.py:416} INFO - Running <TaskInstance: walmart_etl_dag.generate_synthetic_data manual__2025-05-27T05:47:08+00:00 [running]> on host 364c8c3dd6ab
[2025-05-27T05:47:10.140+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='walmart_etl_dag' AIRFLOW_CTX_TASK_ID='generate_synthetic_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T05:47:08+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-27T05:47:08+00:00'
[2025-05-27T05:47:10.145+0000] {logging_mixin.py:154} INFO - Starting synthetic data generation process...
[2025-05-27T05:47:10.145+0000] {logging_mixin.py:154} INFO - Generating supplier data...
[2025-05-27T05:47:10.150+0000] {logging_mixin.py:154} INFO - Found 50 existing supplier records, skipping generation
[2025-05-27T05:47:10.151+0000] {logging_mixin.py:154} INFO - Generating return reason data...
[2025-05-27T05:47:10.151+0000] {logging_mixin.py:154} INFO - Found 10 existing return reason records, skipping generation
[2025-05-27T05:47:10.152+0000] {logging_mixin.py:154} INFO - Transforming and loading data to staging tables...
[2025-05-27T05:47:10.155+0000] {logging_mixin.py:154} INFO - Supplier and return reason data successfully loaded to staging tables with batch ID: BATCH_SYNTHETIC_20250527054710
[2025-05-27T05:47:10.155+0000] {logging_mixin.py:154} INFO - Starting load to target layer for batch BATCH_SYNTHETIC_20250527054710...
[2025-05-27T05:47:10.158+0000] {logging_mixin.py:154} INFO - Supplier and return reason data successfully loaded to target tables
[2025-05-27T05:47:10.158+0000] {logging_mixin.py:154} INFO - Synthetic data generation and loading complete!
[2025-05-27T05:47:10.158+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-27T05:47:10.164+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=walmart_etl_dag, task_id=generate_synthetic_data, execution_date=20250527T054708, start_date=20250527T054710, end_date=20250527T054710
[2025-05-27T05:47:10.186+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-27T05:47:10.203+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
